Frends.Community.AWS.Tests/UploadTests.cs

```diff
@@ -183,7 +183,7 @@ public async Task SendFileToS3Bucket()
             };
 
             var response = await UploadTask.UploadFiles(input, _param, options, new CancellationToken());
-            foreach(var key in response.UploadedFiles)
+            foreach (var key in response.UploadedFiles)
             {
                 Assert.AreEqual("CommunityUploadTest/test.txt", key);
                 await DeleteFileFromBucket(key, _param.BucketName);

```

Comment:
https://github.com/CommunityHiQ/Frends.Community.AWS/blob/fef2efc654cb3383bd049a73b8d3b3d007c4078b/Frends.Community.AWS.Tests/UploadTests.cs#L216
Please use `IsFalse` instead of `IsTrue`.